source/brailleDisplayDrivers/brailleNote.py

```diff
@@ -270,10 +270,10 @@ def display(self, cells):
 			"kb:end": ("br(braillenote):space+d4+d5",),
 			"kb:control+home": ("br(braillenote):space+d1+d2+d3",),
 			"kb:control+end": ("br(braillenote):space+d4+d5+d6",),
-			"kb:enter": ("br(braillenote):space+d8",),
+			"braille_enter": ("br(braillenote):space+d8",),

```

Comment:
Weird. This doesn't even show up in diffs from command line git. I reckon this is a GitHub bogus thingy. :(